packages/material-ui/src/TextareaAutosize/TextareaAutosize.js

```diff
@@ -135,6 +135,7 @@ const TextareaAutosize = React.forwardRef(function TextareaAutosize(props, ref)
       />
       <textarea
         aria-hidden
+        aria-label="Shadow Input"
```

Comment:
What do you think of?

```suggestion
        aria-label="hidden input"
```